core/cloudflow-operator/src/main/scala/cloudflow/operator/action/Action.scala

```diff
@@ -306,7 +309,13 @@ class UpdateStatusAction[T <: ObjectResource](
           editor.updateMetadata(resource, resource.metadata.copy(resourceVersion = existingResource.metadata.resourceVersion))
         )
       res ← resourceVersionUpdated
-        .map(resourceToUpdate => recoverFromConflict(client.updateStatus(resourceToUpdate), client, retries - 1, executeUpdateStatus))
+        .map { resourceToUpdate =>
+          if (predicateForUpdate(existing, resourceToUpdate)) {
+            recoverFromConflict(client.updateStatus(resourceToUpdate), client, retries - 1, executeUpdateStatus)
+          } else {
+            Future.successful(resource)
```

Comment:
Possibly log that this update action was ignored.

Reply:
done